CIP-????/README.md

```diff
@@ -0,0 +1,474 @@
+---
+CIP: ????
+Title: Structured Contracts
+Category: Ledger/Tools
+Status: Active
+Authors:
+  - Polina Vinogradova <polina.vinogradova@iohk.io>
+Implementors: N/A
```

Comment:
```suggestion
Implementors: []
```
I recommend the empty set here since that set is likely to eventually increase.  My understanding of the proposal and ecosystem is that more than one entity could ultimately do this: either themselves or in cooperation.

Reply:
fixed in https://github.com/cardano-foundation/CIPs/pull/963/commits/144c0181ac62c3a5fe8c350d4ba2f7c1e4a9056b